dev/CommonStyles/CommonStyles.vcxitems

```diff
@@ -111,11 +111,29 @@
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)GridViewItem_themeresources.xaml">
+      <UseVisualStyle>LatestWithoutVisualUpdate</UseVisualStyle>
+      <Version>RS1</Version>
+      <Type>ThemeResources</Type>
+      <UseNonstandardCondtionalXAML>true</UseNonstandardCondtionalXAML>
+    </Page>
+    <Page Include="$(MSBuildThisFileDirectory)GridViewItem_themeresources_vu.xaml">
+      <UseVisualStyle>LatestWithVisualUpdate</UseVisualStyle>
+      <Version>21H1</Version>
+      <Type>ThemeResources</Type>
+      <UseNonstandardCondtionalXAML>true</UseNonstandardCondtionalXAML>
+    </Page>
     <Page Include="$(MSBuildThisFileDirectory)ListViewItem_themeresources.xaml">
-      <UseVisualStyle>Latest</UseVisualStyle>
+      <UseVisualStyle>LatestWithoutVisualUpdate</UseVisualStyle>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)ListViewItem_themeresources_vu.xaml">
+      <UseVisualStyle>LatestWithVisualUpdate</UseVisualStyle>
```

Comment:
This new temporary LatestWithVisualUpdate option is used when compiling with a recent insider SDK and then running an app on a recent Windows OS. It uses the _vu suffix.

Reply:
You should not do this. Just follow the existing pattern:
19h1_themeresources.xaml, 21h1_themeresources.xaml

Otherwise it's hard to manage.
generally speaking, everything marked with <Version>21h1</Version> will override the same keys what marked with <Version>RS1</Version>...<Version>19H1</Version

---
In reply to: [561462216](https://github.com/microsoft/microsoft-ui-xaml/pull/3991#discussion_r561462216) [](ancestors = 561462216)